players/group5/player_map.py

```diff
@@ -404,6 +404,25 @@ def get_seen_counts(self, coords: List[List[int]]) -> List[int]:
             seen_counts.append(seen_count)
 
         return seen_counts
+    
+    def get_unseen_counts(self, coords: List[List[int]]) -> int:
+        """
+        Function which returns the number of times each cell has been seen
```

Comment:
@EylamTagor could you update these comments please? What's the difference between seen_count and unseen_count? Is it just counting how many we haven't seen in the given coordinates?